src/module.c

```diff
@@ -168,6 +168,8 @@ typedef struct RedisModuleCtx RedisModuleCtx;
 #define REDISMODULE_CTX_BLOCKED_TIMEOUT (1<<3)
 #define REDISMODULE_CTX_THREAD_SAFE (1<<4)
 #define REDISMODULE_CTX_BLOCKED_DISCONNECTED (1<<5)
+#define REDISMODULE_CTX_TEMP_CLIENT (1<<6)
+#define REDISMODULE_CTX_NEW_CLIENT (1<<7)
```

Comment:
Isn't those two are `not` of each other?

Reply:
no (i was wondering that myself), each of them states how a client should be freed when the context is destroyed.
when neither is set, the client shouldn't be automatically freed.
maybe a comment is due

Reply:
Added a comment